### DISCUSSION TITLE
Hi, I just finished Solidity basic.

### DISCUSSION THREAD

**Discussion by KingVics** [Cyfrin/foundry-full-course-cu#2937] [General]
Hi community, I just finished Solidity beginner course, and I am super excited about my journey to
being a blockchain developer.

**Comment by chainflash** [comment]
congratulations keep going

**Comment by EngrPips** [comment]
Congratulation @KingVics, Keep going don't stop.

**Comment by LordAdhaar** [comment]
Time to relax, eat some popcorn, maybe eat an ice-cream or go to the gym.

Stay froggy.
